doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/autoacknowledge-execution-errors-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='autoacknowledge-execution-errors-proc']
+= Auto-acknowledging execution errors
+
+By default, the execution errors are unacknowledged when they occur. To avoid the need to acknowledge every execution error manually, you can configure jobs to auto-acknowledge some or all execution errors.
+
+[NOTE]
+====
+If you configure an auto-acknowledge job, the job runs every day by default. To auto-acknowledge execution errors only once, set the `SingleRun` parameter to `true`.
+====
+
+.Procedure
+
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Jobs*.
+. In the top right of the screen, click *New Job*.
+. Type any identifier for the job into the *Business Key* field.
+. In the *Type* field, enter the type of the auto-acknowledge job:
+** `org.jbpm.executor.commands.error.JobAutoAckErrorCommand`: Acknowledge all execution errors of type `Job` where the job to which the error relates is now cancelled, completed, or rescheduled for another execution.
+** `org.jbpm.executor.commands.error.TaskAutoAckErrorCommand`: Acknowledge all execution errors of type `Task` where the task to which the error relates is in an exit state (completed, failed, exited, obsolete).
+** `org.jbpm.executor.commands.error.ProcessAutoAckErrorCommand`: Acknowledge all execution errors of any type where the process instance from which the error originates is already finished (completed or aborted), or the task from which the error originates is already finished.
```

Comment:
abort verb
Use only if the term is part of the existing product terminology; for example, a command option or a button
label. In all other instances, replace with "cancel", "stop", "end", or "fail".

Search for in doc.